infrastructures/docker/infrastructure/src/main/java/org/eclipse/che/workspace/infrastructure/docker/server/mapping/ServersMapper.java

```diff
@@ -44,6 +49,17 @@
   public ServersMapper(String hostname, String machineName) {
     this.hostname = hostname;
     this.machineName = machineName;
+    // default to http
+    this.cheHostProtocol = "http";
+  }
+
+  public ServersMapper(
+      @Nullable @Named("che.host.protocol") String cheHostProtocol,
```

Comment:
This parameter is not used in the injection process, so let's not make it more complex by adding an annotation.